src/pages/clouddocs/medical_codes.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Medical Codes"
+excerpt: "."
+categories: Medical Codes
+slug: medical_codes
+toc: true
+---
+<!-- ---
+
+copyright:
+  years: 2019
+lastupdated: "2019-04-16"
+
+keywords: annotator clinical data, clinical data, annotation
+
+subcollection: wh-acd
+
+---
+
+# Medical Codes -->
+
+Annotator for Clinical Data normalizes medical concepts into many common medical codes.  The following table shows the medical code support for each annotator available in Annotator for Clinical Data.
+
```

Comment:
Is there a way to shrink the presentation of the tables so reader doesn't need to scroll off to the right? If not, can we have an indicator so reader knows there's more to scroll through? Not sure that's obvious as presented. I think you have to click into the table to realize it.

Reply:
Unfortunately markdown syntax / rendering takes the largest column and normalizes all columns to that size which directly influences the size of the table.  We could try putting multiple word codes and the header in two text lines vs one to shrink the table size.